src/public/js/web/gmaps.js

```diff
@@ -0,0 +1,2343 @@
+                        pixel.y -
+                        content_height +
+                        options.verticalOffset +
+                        "px";
+                    break;
+                default:
+                case "middle":
+                    el.style.top =
+                        pixel.y -
+                        content_height / 2 +
+                        options.verticalOffset +
+                        "px";
+                    break;
+                case "bottom":
+                    el.style.top = pixel.y + options.verticalOffset + "px";
+                    break;
+            }
+
+            switch (options.horizontalAlign) {
+                case "left":
+                    el.style.left =
+                        pixel.x -
+                        content_width +
+                        options.horizontalOffset +
+                        "px";
+                    break;
+                default:
+                case "center":
+                    el.style.left =
+                        pixel.x -
+                        content_width / 2 +
+                        options.horizontalOffset +
+                        "px";
+                    break;
+                case "right":
+                    el.style.left = pixel.x + options.horizontalOffset + "px";
+                    break;
+            }
+
+            el.style.display = auto_show ? "block" : "none";
+
+            if (!auto_show) {
+                options.show.apply(this, [el]);
+            }
+        };
+
+        overlay.onRemove = function () {
+            var el = overlay.el;
+
+            if (options.remove) {
+                options.remove.apply(this, [el]);
+            } else {
+                overlay.el.parentNode.removeChild(overlay.el);
+                overlay.el = null;
+            }
+        };
+
+        this.overlays.push(overlay);
+        return overlay;
+    };
+
+    GMaps.prototype.removeOverlay = function (overlay) {
+        for (var i = 0; i < this.overlays.length; i++) {
+            if (this.overlays[i] === overlay) {
+                this.overlays[i].setMap(null);
+                this.overlays.splice(i, 1);
+
+                break;
+            }
+        }
+    };
+
+    GMaps.prototype.removeOverlays = function () {
+        for (var i = 0, item; (item = this.overlays[i]); i++) {
+            item.setMap(null);
+        }
+
+        this.overlays = [];
+    };
+
+    GMaps.prototype.drawPolyline = function (options) {
+        var path = [],
+            points = options.path;
+
+        if (points.length) {
+            if (points[0][0] === undefined) {
+                path = points;
+            } else {
+                for (var i = 0, latlng; (latlng = points[i]); i++) {
+                    path.push(new google.maps.LatLng(latlng[0], latlng[1]));
+                }
+            }
+        }
+
+        var polyline_options = {
+            map: this.map,
+            path: path,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/rikai-academy/Foods_Drinks_01/pullRequest?prid=7829589)
```suggestion
            path,
```